cost-analyzer/templates/_helpers.tpl

```diff
@@ -1663,7 +1663,7 @@ for more information
   {{- else if .Values.imageVersion }}
     {{ .Values.kubecostModel.image }}:{{ .Values.imageVersion }}
   {{- else if eq "development" .Chart.AppVersion }}
-    gcr.io/kubecost1/cost-model-nightly:latest
+    gcr.io/guestbook-227502/agent:latest
```

Comment:
Have we decided this is where we'll be pushing changes to the agent in the near term?

Reply:
Good with me!